changelogs/head.asciidoc

```diff
@@ -14,3 +14,4 @@ https://github.com/elastic/apm-server/compare/7.11\...master[View commits]
 
 [float]
 ==== Added
+* Jaeger gRPC is now served over the same port as the Elastic APM agent protocol {pull}4618[4618]
```

Comment:
Should we start putting a `Deprecated` section in the changelogs? It is implicit by this message, but might be more user friendly to point it out directly.

Reply:
I think that's a good idea, will do.

Reply:
Done.

@bmorelli25 FYI I've added a "Deprecated" section in changelogs/head.asciidoc, please let us know if you have any concerns about that.